Game/Source/GDKTestGyms/BenchmarkGymPlayerState.cpp

```diff
@@ -0,0 +1,13 @@
+#include "BenchmarkGymPlayerState.h"
+
+ABenchmarkGymPlayerState::ABenchmarkGymPlayerState()
+{
+
+}
+
+bool ABenchmarkGymPlayerState::ShouldBroadCastWelcomeMessage(bool bExiting)
```

Comment:
```suggestion
bool ABenchmarkGymPlayerState::ShouldBroadCastWelcomeMessage(bool /*bExiting*/)
```

Reply:
bool bExiting /*=  false*/

Reply:
Is there a nice reason to comment out the arg if it is not used?

Reply:
Some compilers will warn if the variable isn't used

Reply:
done